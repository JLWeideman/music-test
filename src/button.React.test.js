import React from "react";
import Button from './Components/SearchBtn'
import Renderer from 'react-test-renderer'

it('Search button rendered correctly', () => {
    const tree = Renderer
        .create(<Button />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})