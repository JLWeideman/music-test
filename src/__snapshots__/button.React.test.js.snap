// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search button rendered correctly 1`] = `
<div
  className="searchOptions"
>
  <button
    onClick={[Function]}
  >
    Search
  </button>
</div>
`;
